### ISSUE TITLE
测试的Visdrone数据集 testloss 一直在63左右震荡

### ISSUE THREAD

**Issue by CharonMing** [YunYang1994/tensorflow-yolov3#401]
__C.TRAIN.BATCH_SIZE = 2
__C.TRAIN.LEARN_RATE_INIT = 1e-4
__C.TRAIN.LEARN_RATE_END = 1e-6
__C.TRAIN.WARMUP_EPOCHS = 2
__C.TRAIN.FISRT_STAGE_EPOCHS = 30
__C.TRAIN.SECOND_STAGE_EPOCHS = 50

**Comment by YunYang1994** [comment]
上个预测效果图？

**Comment by CharonMing** [comment]
> 上个预测效果图？

![sa](https://user-images.githubusercontent.com/55969961/68766119-38061480-0659-11ea-9c4b-8273353ff1c4.png)

**Comment by YunYang1994** [comment]
多少张图片呢，你得讲下你的数据集啊

**Comment by CharonMing** [comment]
> 多少张图片呢，你得讲下你的数据集啊

训练5600张，测试500张左右，2类目标，1920*1080大小的图，又训练40epoch，降到了61左右

**Comment by Byronnar** [comment]
我这个项目是训练Visdrone数据集的，效果比你的好些。
https://github.com/Byronnar/tensorflow-serving-yolov3

**Comment by YunYang1994** [comment]
有Tensorboard吗，把你的 loss curve 贴出来看看

**Comment by CharonMing** [comment]
> 有Tensorboard吗，把你的 loss curve 贴出来看看，每个epoch的训练 loss value 和测试 loss value 是如何变化的，有没有过拟合的危险？并不是训练得越久越好，出现过拟合了可以使用“提前终止”策略。

![捕获1](https://user-images.githubusercontent.com/55969961/68846761-87574e00-0708-11ea-9e34-36b1a53655ca.PNG)
又训了10epoch得到的结果

**Comment by YunYang1994** [comment]
不用一直训练，我记得代码里每训练一个epoch就会输出 test loss，你观察这个值就可以了。

**Comment by CharonMing** [comment]
> 从你给出的结果来看，网络已经收敛了。不用一直训练，我记得代码里每训练一个epoch就会输出 test loss，你观察这个值就可以了。

嗯，选取loss最小的一个模型，测试精度很低

**Comment by YunYang1994** [comment]
batch_size 设置大一点试试，太小了容易过拟合。

**Comment by zhouxianming** [comment]
> > 从你给出的结果来看，网络已经收敛了。不用一直训练，我记得代码里每训练一个epoch就会输出 test loss，你观察这个值就可以了。
> 
> 嗯，选取loss最小的一个模型，测试精度很低

@CharonMing 
你好请问你解决了吗？我的test_loss也有60多，一直下不去，请问增大batch_size后你解决这个问题了吗？

**Comment by CharonMing** [comment]
> 从你给出的结果来看，网络已经收敛了。不用一直训练，我记得代码里每训练一个epoch就会输出 test loss，你观察这个值就可以了。
> 
> 嗯，选取loss最小的一个模型，测试精度很低
> 
> @CharonMing
> 你好请问你解决了吗？我的test_loss也有60多，一直下不去，请问增大batch_size后你解决这个问题了吗？
自己用的数据集和coco数据集有差别，可以生成自己anchors获取或许会减少不少，我的loss降了24左右

**Comment by zhouxianming** [comment]
> > 从你给出的结果来看，网络已经收敛了。不用一直训练，我记得代码里每训练一个epoch就会输出 test loss，你观察这个值就可以了。
> > 嗯，选取loss最小的一个模型，测试精度很低
> > @CharonMing
> > 你好请问你解决了吗？我的test_loss也有60多，一直下不去，请问增大batch_size后你解决这个问题了吗？
> > 自己用的数据集和coco数据集有差别，可以生成自己anchors获取或许会减少不少，我的loss降了24左右

@CharonMing 
请问是根据自己的数据集来生成anchor吗？请问你的loss降到了24左右后测试效果怎么样呢？

**Comment by CharonMing** [comment]
> 从你给出的结果来看，网络已经收敛了。不用一直训练，我记得代码里每训练一个epoch就会输出 test loss，你观察这个值就可以了。
> 嗯，选取loss最小的一个模型，测试精度很低
> @CharonMing
> 你好请问你解决了吗？我的test_loss也有60多，一直下不去，请问增大batch_size后你解决这个问题了吗？
> 自己用的数据集和coco数据集有差别，可以生成自己anchors获取或许会减少不少，我的loss降了24左右
> 
> 
> @CharonMing
> 请问是根据自己的数据集来生成anchor吗？请问你的loss降到了24左右后测试效果怎么样呢？
你可以试试
